policy-controller/src/main.rs

```diff
@@ -10,10 +10,17 @@ use std::net::SocketAddr;
 use structopt::StructOpt;
 use tokio::{sync::watch, time};
 use tracing::{debug, info, info_span, instrument, Instrument};
+use tracing_subscriber::{fmt::format, prelude::*, reload, EnvFilter};
 
 #[derive(Debug, StructOpt)]
 #[structopt(name = "policy", about = "A policy resource prototype")]
 struct Args {
+    #[structopt(parse(from_str = LogLevel::new), long, default_value = "linkerd=info,warn")]
```

Comment:
nit: `structopt` also has [the option to parse an arg from an env var](https://docs.rs/structopt/0.3.23/structopt/#environment-variable-fallback) if it's not present on the command line, so we might want to add
```suggestion
    #[structopt(parse(from_str = LogLevel::new), long, default_value = "linkerd=info,warn", env = "RUST_LOG")]
```